### ISSUE TITLE
[TF 2.0 API Docs] tf.nn.softmax

### ISSUE THREAD

**Issue by prabindh** [tensorflow/tensorflow#33134]
## URL(s) with the issue:

Please provide a link to the documentation entry, for example:
https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/nn/softmax

### Clear description

Yes

### Correct links

yes

### Parameters defined

No. 
Per the code, the first argument "logits" can be of any type that can be passed to "convert_to_tensor()", not just a tensor. Therefore the documentation can be modified to include "Tensor objects, numpy arrays, Python lists, and Python scalars".

### Returns defined
yes

### Raises listed and defined
yes

### Submit a pull request?
yes

**Comment by oanush** [comment]
@prabindh ,
As the PR associated with the issue is merged, can we proceed for the closure ?Thanks!
